src/components/Dashboard/DashboardItem/AnalysisItem.js

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import Link from "next/link";
+import {
+    BarChart,
+    Repeat,
+    Info,
+    Stop,
+    People,
+    PermMedia,
+} from "@material-ui/icons";
+import { IconButton, MenuItem } from "@material-ui/core";
+
+import { formatDate } from "@cyverse-de/ui-lib";
+
+import * as constants from "../constants";
+
+import ItemBase, { ItemAction } from "./ItemBase";
+
+import { getFolderPage } from "../../data/utils";
+
+class AnalysisItem extends ItemBase {
+    constructor({ section, content, height, width }) {
+        super({
+            kind: constants.KIND_ANALYSES,
+            content,
+            section,
+            height,
+            width,
+        });
+    }
+
+    static create(props) {
+        const item = new AnalysisItem(props);
+        return item
+            .addActions([
+                <ItemAction
+                    ariaLabel="relaunch"
```

Comment:
Just mentioning it once, but from what I can tell `ItemAction` is not using the `ariaLabel` prop.  Depending on how you pass in this value, this might need to change to `aria-label`, and I believe they're recommending that aria labels are i18nized.